tasks/getHistoricalData.ts

```diff
@@ -288,28 +286,52 @@ task("getHistoricalData", "Retrieves the historical rates")
             fetch = FETCH_STATUS.ALREADY_FETCHED;
           }
         }
-      } else {
-        console.log("Cannot use borrow flag for Rocket");
       }
 
       // Compound
-      if (taskArgs.compound && !taskArgs.borrow) {
+      if (taskArgs.compound) {
         if (b >= compoundMainnetStartBlock) {
           try {
             if (cToken && decimals) {
-              let r = await cToken.exchangeRateStored({
-                blockTag: b,
-              });
-              if (decimals > 17) {
-                r = r.div(BigNumber.from(10).pow(decimals - 17));
-              } else if (decimals < 17) {
-                r = r.mul(BigNumber.from(10).pow(17 - decimals));
-              }
+              if (taskArgs.borrow) {
```

Comment:
@ioanaroceanu2000 will be helpful to see the apy and rate value plots generated by this script and compare them against compound ui and loanscan historical borrow apy data

Reply:
you can find those comparisons here https://www.notion.so/voltz/Borrowing-Compound-Rate-Oracle-Sanity-Checks-b35fe7889309494a9c453f083cba6002